docs/source/install.ipynb

```diff
@@ -54,27 +64,72 @@
     "conda install -c conda-forge woodwork\n",
     "```\n",
     "\n",
-    ".. note::\n",
-    "    In order to use Woodwork with Dask or Koalas DataFrames, the following commands must be run for your library of choice prior to installing Woodwork with conda: `conda install dask` for Dask or `conda install koalas` and `conda install pyspark` for Koalas.\n",
+    "### Dask\n",
+    "You can use Woodwork with Dask DataFrames by running:\n",
     "\n",
+    "```bash\n",
+    "conda install dask\n",
     "\n",
+    "```\n",
     "\n",
-    "## Source\n",
+    "### Koalas\n",
+    "You can use Woodwork with Koalas DataFrames by running:\n",
     "\n",
-    "To install Woodwork from source, clone the repository from [Github](https://github.com/alteryx/woodwork), and install the dependencies. \n",
+    "```bash\n",
+    "conda install koalas\n",
+    "conda install pyspark\n",
+    "```\n",
+    "\n",
+    "### Update checker\n",
+    "Receive automatic notifications of new Woodwork releases\n",
     "\n",
     "```bash\n",
-    "git clone https://github.com/alteryx/woodwork.git\n",
-    "cd woodwork\n",
-    "python -m pip install .\n",
+    "conda install -c conda-forge alteryx-open-src-update-checker\n",
+    "```\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Scala and Spark\n",
+    "\n",
+    "To use Koalas DataFrames or to run all the unit tests, you will need to install Spark and Scala.\n",
+    "\n",
+    "#### macOS \n",
```

Comment:
@gsheni I think docs are complaining cause these should 3 `###`, and not 4 '####`. Maybe not a big deal

Reply:
Fixed:
https://github.com/alteryx/woodwork/pull/1273/commits/f5622be5a688964d24e31c129baf980841d689fd